tests/performance_tests/test_memory_usage.py

```diff
@@ -62,45 +61,38 @@ def mock_target_accessor() -> EnsembleAccessor:
 
 
 def make_source_accessor(path: Path, ert: EnKFMain) -> EnsembleReader:
-    realisations = ert.getEnsembleSize()
-    uuid = uuid4()
-    path = Path(path) / "ensembles" / str(uuid)
-    source = LocalEnsembleAccessor.create(
-        None,
-        path,
-        uuid,
-        ensemble_size=realisations,
-        experiment_id=uuid4(),
-        iteration=1,
-        name="default",
-        prior_ensemble_id=None,
-    )
-
-    ens_config = ert.ensembleConfig()
-    obs_keys = ert.getObservations().getTypedKeylist(
-        EnkfObservationImplementationType.GEN_OBS
-    )
-    observations = ert.getObservations()
-
-    obs_data_keys = ens_config.getKeylistFromImplType(SummaryConfig)
-    for real in range(realisations):
-        for obs_key in obs_keys:
-            obs = observations[obs_key]
-            obs_vec = obs.observations[0]  # Ignores all other time points for now
-            obs_highest_index_used = obs_vec.getDataIndex(len(obs_vec) - 1)
+    path = Path(path) / "ensembles"
+    with open_storage(path, mode="w") as storage:
+        experiment_id = storage.create_experiment(
+            parameters=ert.ensembleConfig().parameter_configuration
+        )
+        source = storage.create_ensemble(experiment_id, name="prior", ensemble_size=100)
+        gen_obs_keys = ert.getObservations().getTypedKeylist(
+            EnkfObservationImplementationType.GEN_OBS
+        )
+        observations = ert.getObservations()
+
+        ens_config = ert.ensembleConfig()
```

Comment:
Might as well get+store this variable just below L65 to remove the dbl invocation

Reply:
Done